harvestingkit/inspire_cds_package/mappings.py

```diff
@@ -366,6 +366,10 @@
       {
         "cds": "Nuclear Physics - Theory",
         "inspire": "Theory-Nucl"
+      },
+      {
+        "cds": "Quantum Technology",
+        "inspire": "Quantum Physics"
```

Comment:
FYI, we've also added `Condensed Matter` to INSPIRE, but that already exists on CDS. I don't think you'd need to add it here though thanks to https://github.com/CERNDocumentServer/harvesting-kit/blob/8d102d5f3b85c87d57f06f667abcf91cdaf3de4f/harvestingkit/inspire_cds_package/base.py#L97.